spec/db.spec.js

```diff
@@ -144,6 +151,41 @@ describe('db', () => {
           });
         });
 
+        describe('.getTableCreateScript', () => {
+          it('should return table create script', async() => {
+            const [createScript] = await dbConn.getTableCreateScript('users');
+
+            if (dbClient === 'mysql') {
+              expect(createScript).to.contain('CREATE TABLE `users` (\n' +
```

Comment:
I can, but than the code won't be indented, since it will count tab spaces as a part of the template string, so the test will fail. I find that way less readable, since it's moved to the left, but if others think differently I'll change it, no problem :+1: